experimental/platform_data/defs.bzl

```diff
@@ -0,0 +1,84 @@
+"""
+The platform_data rule can be used to change the target platform of a target,
+and then depend on that elsewhere in the build tree. For example:
+
+cc_binary(name = "foo")
```

Comment:
One last nit: add `load("@platforms//experimental/platform_data:defs.bzl", "platform_data")` to this example so that users can more easily copy and paste it in.

Reply:
Done. Thank you!